Misc/NEWS.d/next/Core_and_Builtins/2024-07-25-01-45-21.gh-issue-122239.7zh-sW.rst

```diff
@@ -0,0 +1,2 @@
+Improve the error message for unbalanced unpacking, showing the
```

Comment:
You should mention the new exception and document it as well.

Reply:
The place to document it is `Doc/library/exceptions.rst`.

Reply:
If the approach seems good, I can get started on documentation.